argus/backend/plugins/sct/testrun.py

```diff
@@ -215,7 +216,8 @@ def get_events(self) -> list[EventsBySeverity]:
         return self.events
 
     def submit_product_version(self, version: str):
-        self.scylla_version = version
+        if not self.version_source:
```

Comment:
so basically we ignore if version_source is set?

Reply:
Yes.